containerd/driver.go

```diff
@@ -358,6 +358,10 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 	// Setup environment variables.
 	var env []string
 	for key, val := range cfg.Env {
+		// Don't override $PATH.
```

Comment:
Let's setup a future proof thing here for any keys that need to be other than PATH and have it default to just PATH

Reply:
https://play.golang.org/p/v5gGhQQ-9_7

Reply:
Fixed.